### PR TITLE
BI-30: TruongHung - Implement and update for print out an interesting square with O, +, -

### PR DESCRIPTION
**Pull request by TruongHung**
## Exercise

* [Exercise-xx](https://github.com/NT-Line-team/Behave_test/issues/<issue_number>)

## Overview

Write an overview of what work was done and how, also mention any hurdles or complications.  Include links to additional information, articles, and documentation that might be useful to the reviewer.

## Verification Steps

1. `git checkout pr/###`
2. Check the functionality

### Docs
- [ ] Are changes required for docs?
    - [ ] `README.md` was updated?
    - [ ] Additional documentation required for `docs/`?
- [ ] No Doc Changes.

### CODE REVIEW

File: Behave_test-develop/EricTruong30.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            j1='-'
+        if(j==0 or j== n or j==i):
+            j1 = 0
+        if (i==0 or i== n-1):
+            j1 = 0
+        print (j1),
+    print(0)
+#print i
+
+
```
Comment (hungphamviet) [inline Behave_test-develop/EricTruong30.py]: * Create your own directory in `python_basic/truonghung`, then put your code there.
* Name your python file for being more informative. For example, `EricTruong30.py` might be renamed to print_square.py`
* Refine commit's message, pull request's summary, description. Only one reviewer should be granted.

Comment (TruongHung) [inline Behave_test-develop/EricTruong30.py]: Hi A @hungphamviet ,
I updated base on your feedback, please help me check it
Best wishes,
Q.Hung

**Comment by hungphamviet** [comment]
Please also refine PR summary as our conventions.

**Comment by hungphamviet** [comment]
This is the result from my side:
```
input a number :15
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 - - - - - - - - - - - - 0
0 + 0 - - - - - - - - - - - 0
0 + + 0 - - - - - - - - - - 0
0 + + + 0 - - - - - - - - - 0
0 + + + + 0 - - - - - - - - 0
0 + + + + + 0 - - - - - - - 0
0 + + + + + + 0 - - - - - - 0
0 + + + + + + + 0 - - - - - 0
0 + + + + + + + + 0 - - - - 0
0 + + + + + + + + + 0 - - - 0
0 + + + + + + + + + + 0 - - 0
0 + + + + + + + + + + + 0 - 0
0 + + + + + + + + + + + + 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
```
The result looks good. I will merge code once my remaining comments are reviewed and updated. Thanks.

**Comment by TruongHung** [comment]
Thank for your feedback, i'm going do other exercise

**Pull request merged**